src/lib/containers/entity_finder/details/EntityDetailsList.tsx

```diff
@@ -141,5 +142,5 @@ export const EntityDetailsList: React.FunctionComponent<
     setComponent(getComponentFromConfiguration(configuration))
   }, [configuration, sharedProps])
 
-  return component
+  return <SynapseErrorBoundary>{component}</SynapseErrorBoundary>
```

Comment:
There are edge cases that just break the right pane of the entity finder. Don't need to fix them today (they are hard states to get into, e.g. navigating to a folder where you don't have VIEW access). But this will make them easier to deal with in development.